protocol/communities/manager.go

```diff
@@ -4277,6 +4277,20 @@ func (m *Manager) GetOwnedCommunitiesChatIDs() (map[string]bool, error) {
 	return chatIDs, nil
 }
 
+func (m *Manager) GetOwnedCommunitiesUniversalChatIDs() (map[string]bool, error) {
+	ownedCommunities, err := m.Controlled()
+	if err != nil {
+		return nil, err
+	}
+	chatIDs := make(map[string]bool)
+	for _, c := range ownedCommunities {
+		if c.Joined() {
+			chatIDs[c.UniversalChatID()] = true
```

Comment:
nothing specific, you are right it can be changed to empty struct. 

I just copied function `GetOwnedCommunitiesChatIDs` from above and did not think about it.

Reply:
looks like it is used in caller in a condition and hence bool makes it easier and cleaner to look at. 

I don't think there is any scenario for false. 

So leaving it as is.